src/Nethermind/Nethermind.State/IWorldStateManager.cs

```diff
@@ -11,6 +11,7 @@ public interface IWorldStateManager
     IWorldState GlobalWorldState { get; }
     IStateReader GlobalStateReader { get; }
     IReadOnlyTrieStore TrieStore { get; }
+    bool SupportHashLookup { get; }
```

Comment:
Not sure if needed as we already expose `TrieStore` and it already kind-of exposes `Scheme`, so maybe extension method would suffice? Or default implementation?

Reply:
Ideally, we should not expose triestore at all. Its just a bit.. un-nice at the moment.